### PR TITLE
 Bump vcpkg to recent commit  and remove custom ports

### PR DESCRIPTION
**Pull request by traversaro**
As the ogre ( microsoft/vcpkg#14006 ) and portaudio ( microsoft/vcpkg#13982 ) problems were fixed, we can remove the custom ports. 

I also added  a few `vcpkg --list` calls to easily provide the details on the version to be added in the "Detailed Info" of the Release description such as in https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/releases/tag/v0.4.0 .

### CODE REVIEW

**Comment by traversaro** [comment]
It could make sense to wait for these two related fixes, and then bump vcpkg again:
* https://github.com/microsoft/vcpkg/issues/14045

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ jobs:
       run: |
         # TinyXML is not installed as a workaround for https://github.com/robotology/ycm/pull/296
         C:/robotology/vcpkg/vcpkg.exe --overlay-ports=C:/robotology/robotology-vcpkg-ports --overlay-ports=${GITHUB_WORKSPACE}/custom-ports install --triplet x64-windows ace asio boost-asio boost-bind boost-process boost-dll boost-filesystem boost-system freeglut gsl eigen3 esdcan-binary glew glfw3 ode openssl libxml2 libjpeg-turbo eigen3 opencv portaudio matio sdl1 sdl2 qt5-base[latest] ipopt-binary qt5-declarative qt5-multimedia qt5-quickcontrols qt5-quickcontrols2 sqlite3[core,tool]
-
+        C:/robotology/vcpkg/vcpkg.exe list
```
Comment (S-Dafarra) [inline .github/workflows/ci.yml:46]: Is this like a debug print?

Comment (traversaro) [reply .github/workflows/ci.yml:46]: Sorry, this was added intentionally to provide the details on the version to be added in the "Detailed Info" of the Release description such as in https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/releases/tag/v0.4.0 . I updated the PR description.

Comment (S-Dafarra) [reply .github/workflows/ci.yml:46]: Ah, I see!

**Review by S-Dafarra — APPROVED** [review]

**Comment by traversaro** [comment]
> 
> 
> It could make sense to wait for these two related fixes, and then bump vcpkg again:
> 
>     * [microsoft/vcpkg#14045](https://github.com/microsoft/vcpkg/issues/14045)

This has been merged, so we can now bump vcpkg to https://github.com/microsoft/vcpkg/commit/70f192e073dd9e5ed503fc1e86d8a54e2fbaceb4 .

**Pull request merged**